main.go

```diff
@@ -676,3 +676,18 @@ func parseUnixOrTCPAddress(input string) (network, address, host string, err err
 	network, address = "tcp", input
 	return
 }
+
+func proxyLoggerFlags(flags []string) int {
```

Comment:
It took a while to understand what's happening here. A quick comment about silencing particular log types might be helpful here.

Reply:
Can do